winpr/libwinpr/crypto/cipher.c

```diff
@@ -269,122 +252,6 @@ static const EVP_CIPHER* winpr_openssl_get_evp_cipher(int cipher)
 		case WINPR_CIPHER_AES_256_CCM:
 			evp = EVP_get_cipherbyname("aes-256-ccm");
 			break;
-
```

Comment:
why remove these?

Reply:
Same reason for removing the deprecated cipher IDs we map with mbedtls in the headers, this is just another mapping function